django/librarian/templates/librarian/components/document_status.html

```diff
@@ -109,8 +109,8 @@ <h6>{% trans "Document status" %}</h6>
           {{ document.num_chunks }}
         </dd>
       {% endif %}
-      {% if document.cost %}
-        <dt>{% trans "Processing cost" %}</dt>
+      {% if document.usd_cost %}
+        <dt>{% trans "Processing cost (cumulative)" %}</dt>
```

Comment:
Because of the way costs are calculated / set for the documents, only cumulative cost is really possible now.